src/content/docs/journey-large-logs/organize.mdx

```diff
@@ -0,0 +1,196 @@
+
+New Relic provides the ability to group these remaining logs into thematic partitions and parse the logs to pull out and tag valuable attributes. By organizing your logs in this manner, you can:
+
+* Query for any attribute within your logs
+* Manage specific partitions at a time, such as front-end logs vs back-end logs
+* Reduce query load times 
+
+
+## Why partition logs
+
+Seperating your logs into discrete partitions provides both performance benefits and management benefits. For example, imagine you want to query for an attribute only found in front-end logs. Querying through 5 TB of logs it will take drastically longer than querying through a specific 1 TB partition of only front-end logs. 
+
+The goals of partitioning your data should be:
+*   Create data partitions that align with concepts in your environment or organization that are static or change infrequently (for example, by business unit, team, environment, service, etc.).
+*  Ensure each partition remains below 1 TB of daily ingest for optimal performance.
+
+There are two types of retention for partitions. Choose **Standard** for maximimum retention; use this for valuable data you may want to query far in the future. Choose **Secondary** for 30-day retention; use this for less valuable data or for data you will only need in the near future. 
+
+## Why parse logs
+
+Parsing your logs at ingest is the best way to make your log data more usable by you and other users in your organization. For example, compare these two logs pre-parsing and post-parsing using a Grok parsing rule:
+
+<SideBySide>
+    <Side>
+Pre-parsing:
+
+```
+{
+    "message": "32 4329 store_Portland"
+}
+
+
+```
+
+    </Side>
+    <Side>
+
+Post-parsing:
+
+```
+{
+    "transaction_total": "32",
+    "purchase_number": "4329",
+    "store_location": "store_Portland",
+}
+```
+
+    </Side>
+</SideBySide>
+
+
+
+This allows you to easily query for newly defined attributes such as `transaction_total` in dashboards and alerts.
+
+
+
+## Organize your logs
+
+The following steps will guide you through how to organize your logs in the New Relic UI.
+
+<Steps>
+    <Step>
+    ### Navigate to the UI
+
+    Go to **[one.newrelic.com > Logs](https://one.newrelic.com/launcher/logger.log-launcher)**    
+    </Step>
+    <Step>
+
+### Partition your logs
+
+Let's say ACME Corp knows they will ingest about 2 TBs of logs in the coming months. They want to create partitions for logs coming from their Java app and from their infrastructure agent. They think they may need to query their Java logs far into the future so they decide to use a standard retention. Meanwhile, they only need recent infrastrucutre logs so they will use secondary retention for those.
+
+To do so:
+
+<SideBySide>
+    <Side>
+
+1. From **Manage data** on the left nav, click **Data partitions**, then click **Create partition rule**.
+2. Define a Partition name as an alphanumeric string that begins with Log_. In this case `Log_java`. 
```

Comment:
why is `Partition` capitalized here? is this UI text? if so, should be bold.

Reply:
> that begins with Log_.

`Log_` should be wrapped in code, esp next to a period here where the period introduces syntactic ambiguity.